### PR TITLE
1.13 Particles and Potion Effects

### PR DESCRIPTION
**Pull request by ShaneBeee**
Target Minecraft versions: 1.13.x
Requirements: none
Related issues: #1385 

Description:
Added in the new particles and potion/status effects from 1.13.x

### CODE REVIEW

File: src/main/java/ch/njol/skript/util/VisualEffect.java [hunk 1]
```diff
@@ -184,7 +184,16 @@ else if (raw instanceof ItemType) {
 		
 		// 1.11 particles
 		TOTEM("TOTEM"),
-		SPIT("SPIT");
+		SPIT("SPIT"),
+		
+		// 1.13 particles
+		SQUID_INK("SQUID INK"),
+		BUBBLE_POP("BUBBLE POP"),
```
Comment (TheBentoBox) [inline src/main/java/ch/njol/skript/util/VisualEffect.java]: Shouldn't these be `SQUID_INK("SQUID_INK")` and `BUBBLE_POP("BUBBLE_POP")`?

File: src/main/resources/lang/english.lang [hunk 2]
```diff
@@ -1246,34 +1269,34 @@ inventory types:
...
+	ocelot_baby: ocelot baby
+	reinforcements: reinforcements
+	shoulder_entity: perching, shoulder
+	silverfish_block: silverfish reveal, silverfish trap
+	slime_split: slime split
+	spawner: mob spawner, creature spawner, spawner
+	spawner_egg: spawn egg
+	trap: trap
+	village_defense: village defense, golem defense, iron golem defense
+	village_invasion: village invasion, village invading
```
Comment (TheBentoBox) [inline src/main/resources/lang/english.lang]: I guess this explains why I couldn't use the spawn reasons last night.

File: src/main/java/ch/njol/skript/util/VisualEffect.java (hunk 1 above)
Comment (bensku) [reply src/main/java/ch/njol/skript/util/VisualEffect.java]: Yes.

Comment (ShaneBeee) [reply src/main/java/ch/njol/skript/util/VisualEffect.java]: Ah yes my bad, this has been fixed!

**Comment by Nicofisi** [comment]
Big thanks to both of you! <3

**Pull request merged**